README.md

```diff
@@ -18,23 +18,35 @@
 
 ## Charts
 
-Please look in the chart directories for the documentation for each chart. These
-Helm charts are designed to be a lightweight way to configure our official
+These Helm charts are designed to be a lightweight way to configure our official
 Docker images. Links to the relevant Docker image documentation has also been
 added below.
 
-| Chart                                      | Docker documentation                                                            |
-|--------------------------------------------|---------------------------------------------------------------------------------|
-| [APM-Server](./apm-server/README.md)       | https://www.elastic.co/guide/en/apm/server/current/running-on-docker.html       |
-| [Elasticsearch](./elasticsearch/README.md) | https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html     |
-| [Filebeat](./filebeat/README.md)           | https://www.elastic.co/guide/en/beats/filebeat/current/running-on-docker.html   |
-| [Kibana](./kibana/README.md)               | https://www.elastic.co/guide/en/kibana/current/docker.html                      |
-| [Logstash](./logstash/README.md)           | https://www.elastic.co/guide/en/logstash/current/docker.html                    |
-| [Metricbeat](./metricbeat/README.md)       | https://www.elastic.co/guide/en/beats/metricbeat/current/running-on-docker.html |
+We recommend that the Helm chart version is aligned to the version of the product
+you want to deploy. This will ensure that you using a chart version that has been
+tested against the corresponding production version.  
+This will also ensure that the documentation and examples for the chart will work 
+with the version of the product you are installing.
+
+For example if you want to deploy an Elasticsearch `7.5.1` cluster, use the
+corresponding `7.5.1` [tag][elasticsearch-751].
```

Comment:
I would suggest using a more recent tag like https://github.com/elastic/helm-charts/tree/7.7.1/elasticsearch/ which provides better documentation.

Reply:
Done in 502cc2d